app/gen-server/migration/1759434763338-DocDisabledAt.ts

```diff
@@ -0,0 +1,20 @@
+import {nativeValues} from "app/gen-server/lib/values";
+import {MigrationInterface, QueryRunner, TableColumn, TableIndex} from "typeorm";
+
+export class DocDisabledAt1759434763338 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+      await queryRunner.addColumn("docs", new TableColumn({
+        name: "disabled_at",
+        type: nativeValues.dateTimeType,
+        isNullable: true,
+      }));
+      await queryRunner.createIndex("docs", new TableIndex({
```

Comment:
What is the purpose of this index?

Reply:
We have several queries where `disabled_at = NULL` in SQL, so the index seems necessary. The corresponding `removed_at` column also has an index and the queries for the two properties are very similar.